Bresenham_Circle_Drawing_Algo/bresenham_circle_algo.c

```diff
@@ -0,0 +1,55 @@
+#include<graphics.h>
+#include<stdlib.h>
+#include<stdio.h>
+#include<conio.h>
+void main()
+{
+    clrscr();
+
+    //r is the radius of the circle
+    //x0 and y0 is the centre of the circle
+    //x and y are an arbitrary point on circle
+    //p is the decision parameter
+    int x,y,x0,y0,p,r;
+    int gdriver = DETECT, gmode;
+    initgraph(&gdriver,&gmode, "");
+
+    printf("Enter the radius of circle.");
+    scanf("%d",&r);
+    printf("Enter co-ordinates of centre of circle.");
+    scanf("%d%d",&x0,&y0);
+
+    x=0;
+    y=r;
+    //putpixel is used to plot the point
+    putpixel(x0+x,y0-y,10);
+
+    p=3-(2*r);
+    while(x<=y)
+   {
+        if(p<0)
+            {p=p+4*x+6;}
+        else
+            {p=p+4*(x-y)+10;
```

Comment:
Keep the parenthesis of each block in separate line just like you did for the opening parenthesis of while loop.
All binary operators except ` . ` should be separated from their operands by spaces. Blank spaces should never separate unary operators such as unary minus, increment ("++"), and decrement ("--") from their operands.